lib/loklak.rb

```diff
@@ -1,11 +1,12 @@
-require "net/http"
-require "json"
-require "loklak/version"
+require 'net/http'
+require 'json'
+require 'loklak/version'
 
-BASE_API_URL = "http://loklak.org/api/"
+BASE_API_URL = 'http://loklak.org/api/'.freeze
```

Comment:
can you explain why freeze is necessary here as well as in line no.2 in version.rb ? (doubt)